openapi/update.sh

```diff
@@ -44,11 +44,15 @@ transform_schema() {
             # Delete enumerated error code cases per above.
             elif .key | test("^4\\d\\d$") then
                 empty
-            # Convert base64-encoded objects to bare strings, as the former
-            # is not technically in the OpenAPI spec.
-            # https://spec.openapis.org/registry/format/byte.html
-            elif .key == "attestation" and .value.type? == "object" then
+            # LogEntry.body is incorrectly marked as an object in the current
+            # schema; adjust to a string. Technically the same condition as
+            # below, but separate for easy removal when fixed upstream.
+            # https://github.com/sigstore/rekor/pull/2091
+            elif .key == "body" and .value.type? == "object" then
                 .value = {"type": "string"}
+            # Fix missing inner type on LogEntry.attestation.data.
+            elif .key == "attestation" and .value.type? == "object" then
+                .value = {"type": "object", "properties": { "data": {"type": "string", "format": "byte"}}}
```

Comment:
This applies two fixes:

* `LogEntry.body` is marked as a JSON object in the upstream schema, but it's really a (base64) string.
* The previous patch for `LogEntry.attestation` was slightly wrong, but went un-noticed since relatively few entries have an attestation: `attestation` is an object not a string, but its inner `data` member is a bas64 string.